src/drivers/actuators/voxl_esc/voxl_esc.cpp

```diff
@@ -1047,20 +1077,35 @@ bool VoxlEsc::updateOutputs(bool stop_motors, uint16_t outputs[MAX_ACTUATORS],
 			}
 		}
 	}
+	
 
 	Command cmd;
-	cmd.len = qc_esc_create_rpm_packet4_fb(_esc_chans[0].rate_req,
-					       _esc_chans[1].rate_req,
-					       _esc_chans[2].rate_req,
-					       _esc_chans[3].rate_req,
-					       _esc_chans[0].led,
-					       _esc_chans[1].led,
-					       _esc_chans[2].led,
-					       _esc_chans[3].led,
-					       _fb_idx,
-					       cmd.buf,
-					       sizeof(cmd.buf));
-
+	if (_extended_rpm) {
+		cmd.len = qc_esc_create_rpm_div2_packet4_fb(_esc_chans[0].rate_req,
```

Comment:
Since you have an extended_rpm flag maybe you can just have the single function and you pass this flag to indicate it has extended rpm and can use the new packet type.

Reply:
I think this is a good idea... it will reduce the code in voxl_esc.cpp as well. I will get this going!